rosco-web/config/rosco.yml

```diff
@@ -4,7 +4,7 @@ server:
 rush:
   baseUrl: http://localhost:8085
   # TODO(duftler): Figure out the right way to set this dynamically.
-  configDir: /some/path/to/config/packer
+  configDir: rosco-web/config/packer
```

Comment:
Did this change make things work for you? Pretty sure that this config value is used in building the command to rush. And then rush will try to resolve that path, not rosco.